addons/mail/__manifest__.py

```diff
@@ -216,6 +218,7 @@
         'web.tests_assets': [
             'mail/static/tests/helpers/**/*.js',
             'mail/static/tests/models/*.js',
+            'mail/static/src/models_data/*.js',
```

Comment:
We don't actually need `models_data` in test. Qunit tests have a much smaller sample of emojis (see `emoji_registry_qunit_tests.js`)